.husky/pre-commit

```diff
@@ -4,6 +4,13 @@
 cd contracts
 yarn lint-staged
 
-cd ../clients/geth/specular
-make install
-echo "Successfully installed Specular"
+GO_DIRECTORY="../clients/geth/specular"
+if git diff --cached --quiet -- "$GO_DIRECTORY"; then
```

Comment:
- where's GO_DIRECTORY defined?
- for --quiet don't you need to check the value of `$?` to see what was returned?